content/docs/machine-learning/creating-a-new-kubeflow-notebook.md

```diff
@@ -0,0 +1,103 @@
+
+- **RStudio**, the first party development IDE and ecosystem for the R language. If you’re someone who prefers R over Python, this is the choice for you!
+
+:::tipCustom environments
+If you can’t find what you’re looking for or are the type of person who brings their own IDE, you’re also welcome to pull a custom container. Please ensure they meet the Kubeflow container image requirements [listed here](https://www.kubeflow.org/docs/components/notebooks/container-images/#image-requirements).
+:::
+
+### CPU / RAM
+
+When creating a new notebook, you must specify the resources you want it to use in your cluster. The fields are:
+
+- Minimum CPU: The scale-down utilisation of your notebook in CPU cores
+- Maximum CPU: The scale up utilisation of your notebook in CPU cores, up to the core limit of your chosen Kubeflow cluster size (by default this is 120% of minimum CPU)
+- Minimum Memory Gi: The minimum memory (in Gibibytes) your notebook can utilise
+- Maximum Memory Gi: the maximum memory (in Gibibytes) your notebook can utilise, up to the RAM limit of your chosen Kubeflow cluster size (by default this is 120% of minimum memory)
+
+### GPUs
+
+:::warningGPU support coming soon
+Please leave all GPU-related fields on the notebook setup page blank to prevent issues with your notebook server being created. Kubeflow will also warn you if you try to add a GPU.
+:::
+
+The Civo machine learning Kubeflow service runs a standard Kubeflow installation on your cluster, displaying all Kubeflow notebook option fields.
+
+GPUs are not yet available on Civo, meaning that though GPU options are displayed, any values entered into these fields will prevent the notebook from starting.
+
+GPU support is under development and this documentation will be updated to reflect the available options.
+
+### Workspace volume
+
+You can specify a workspace volume (new or existing) to be mounted as a PVC Volume on your home directory.
+
+:::tipPersistent data volumes
+A StatefulSet assigns unique identifiers to each pod and allows you to easily store and track data in a persistent data volume, however, an important thing to remeber is that this is a completely separate entity and has an independent lifecycle from the pods. The persistent volume is a cluster resource like a node inside the cluster. This abstracts out a lot of details on how storage is provided from how it is consumed, it retrieves data needed for analysis from the main storage and writes back changes as needed
+
+The persistent data volume is connected to a particular pod by a persistent volume claim, these allow a user to consume abstract storage resources. In the instance that you delete the notebook server, the data still persists in the persistent data volume assigned to that pod and if a new notebook server is created, it would be able to access the same data from the same persistent data volume.
```

Comment:
Should be

The persistent data volume is connected to a particular pod by a Persistent Volume Claim (PVC). These allow a user to consume abstract storage resources...